src/libm/sleefsimdsp.c

```diff
@@ -2472,6 +2965,77 @@ EXPORT CONST vfloat xerfcf_u15(vfloat a) {
   r = vsel_vf_vo_vf_vf(visnan_vo_vf(s), vcast_vf_f(SLEEF_NANf), r);
   return r;
 }
+#endif // #if !defined(DETERMINISTIC)
+
+#if !defined(DETERMINISTIC) && !defined(ENABLE_GNUABI)
+
+#ifdef ENABLE_ALIAS
```

Comment:
Can you explain what are these alias for? Please do it in a comment in the code.